exporter/logicmonitorexporter/factory_test.go

```diff
@@ -38,99 +40,55 @@ func TestCreateDefaultConfig(t *testing.T) {
 	assert.NoError(t, componenttest.CheckConfigStruct(cfg))
 }
 
-// TODO: Lines commented out until implementation is available
-// func TestCreateTracesExporter(t *testing.T) {
-// 	tests := []struct {
-// 		name    string
-// 		config  Config
-// 		wantErr bool
-// 	}{
-// 		{
-// 			name: "Error scenario",
-// 			config: Config{
-// 				HTTPClientSettings: confighttp.HTTPClientSettings{
-// 					Endpoint: "example.com",
-// 				},
-// 			},
-// 			wantErr: true,
-// 		},
-// 		{
-// 			name: "Non Error scenario",
-// 			config: Config{
-// 				ExporterSettings: config.NewExporterSettings(component.NewID(typeStr)),
-// 				HTTPClientSettings: confighttp.HTTPClientSettings{
-// 					Endpoint: "http://example.com",
-// 					TLSSetting: configtls.TLSClientSetting{
-// 						Insecure: false,
-// 					},
-// 				},
-// 			},
-// 			wantErr: false,
-// 		},
-// 	}
+func TestCreateLogsExporter(t *testing.T) {
+	tests := []struct {
+		name         string
+		config       Config
+		shouldError  bool
+		errorMessage string
+	}{
+		{
+			name: "valid config",
+			config: Config{
+				ExporterSettings: config.NewExporterSettings(component.NewID(typeStr)),
+				HTTPClientSettings: confighttp.HTTPClientSettings{
+					Endpoint: "http://example.logicmonitor.com/rest",
+				},
+			},
+			shouldError: false,
+		},
+		// {
```

Comment:
Please don't commit commented out code, it just builds clutter.

Reply:
sure will remove commented blocks.